gradle-conjure-api/src/main/java/com/palantir/gradle/conjure/api/EndpointVersion.java

```diff
@@ -31,6 +31,9 @@
     @JsonProperty("min-version")
     private String minVersion;
 
+    @JsonProperty("max-version")
```

Comment:
Do you have tests for how reading in old data (without the max?) works?  Does this need to be marked as optional? Or have the build auto-insert "x.x.x" or something?